mnefun/_mnefun.py

```diff
@@ -995,19 +998,47 @@ def run_sss_locally(p, subjects, run_indices):
                 coord_frame='head', head_pos=pos, regularize=reg)
 
             raw_sss.save(o, overwrite=True, buffer_size_sec=None)
-            #  process erm files if any
-            for ii, (r, o) in enumerate(zip(erm_files, erm_files_out)):
-                if not op.isfile(r):
-                    raise NameError('File not found (' + r + ')')
-                raw = Raw(r, preload=True, allow_maxshield=True)
-                raw.load_bad_channels(prebad_file, force=True)
-                # apply maxwell filter
-                raw_sss = maxwell_filter(
-                    raw, int_order=p.int_order, ext_order=p.ext_order,
-                    calibration=cal_file, cross_talk=ct_file,
-                    st_correlation=p.st_correlation, st_duration=st_duration,
-                    destination=None, coord_frame='meg')
-                raw_sss.save(o, overwrite=True, buffer_size_sec=None)
```

Comment:
@kambysese this loop was at the wrong level so I think these files were being processed many times more than they needed to be